opf/anomalyTester.py

```diff
@@ -1,97 +0,0 @@
-#!/usr/bin/env python
-
-from nupic.frameworks.opf.modelfactory import ModelFactory
```

Comment:
this is complicated :) I've first merged functionality of anomaly_benchmark into anomalyTester, removed the benchmark, and then renamed anomalyTEster to anomaly_benchmark (to increase the confusion). So please see the file below.